common/app/model/content.scala

```diff
@@ -452,6 +453,7 @@ final case class Article (
   private lazy val soupedBody = Jsoup.parseBodyFragment(fields.body)
   lazy val hasKeyEvents: Boolean = soupedBody.body().select(".is-key-event").nonEmpty
   lazy val isSport: Boolean = tags.tags.exists(_.id == "sport/sport")
+  //@deprecated("use content.blocks", "")
```

Comment:
I want it to be deprecated for future use, but adding a deprecated annotation stops any old usage of it from compiling.  I just wanted it to be a warning but I couldn't do that.